sdk/python/resources/workspace/workspace-managed-network.ipynb

```diff
@@ -0,0 +1,366 @@
+   "metadata": {
+    "name": "my_ml_client"
+   },
+   "outputs": [],
+   "source": [
+    "ws_provisioned = ml_client.workspaces.begin_provision_network(\n",
+    "    workspace_name=mvnet_workspace_name\n",
+    ").result()\n",
+    "\n",
+    "print(ws_provisioned)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 3. Adding outbound rules to the workspace\n",
+    "Outbound rules can be added to the workspace's managed network to allow outbound communication from the network. The possible types are `ServiceTag`, `FQDN`, or `PrivateEndpoint`.\n",
+    "\n",
+    "In this example, we will include all three types for demonstration purposes."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 3.1 Add a ServiceTag outbound rule\n",
+    "To create a ServiceTag outbound rule, we will define the `ServiceTagDestination` Python object with the following attributes:\n",
+    "- `name` - Name of the ServiceTag outbound rule.\n",
+    "- `service_tag` - Service Tag of an Azure service that maps to predefined IP addresses for its service endpoints.\n",
+    "- `protocol` - Allowed transport protocol, can be \"TCP\", \"UDP\", \"ICMP\" or \"*\" for all supported protocols.\n",
+    "- `port_ranges` - A comma-separated list of single ports and/or range of ports, such as 80,1024-65535 that traffics are allowed.\n",
+    "\n",
+    "To see available Azure Service tags, you can reference [Azure Virtual Network Service Tags documentation](https://learn.microsoft.com/en-us/azure/virtual-network/service-tags-overview#available-service-tags)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ws_mvnet = ml_client.workspaces.get(mvnet_workspace_name)\n",
+    "\n",
+    "ws_mvnet.managed_network.outbound_rules.append(\n",
+    "    ServiceTagDestination(\n",
+    "        name=\"servicetagrule\",\n",
+    "        service_tag=\"AzureCloud\",\n",
+    "        protocol=\"TCP\",\n",
+    "        port_ranges=\"80, 8080\",\n",
+    "    )\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 3.2 Add a FQDN outbound rule\n",
+    "Note: FQDN outbound rules are implemented using Azure Firewall. If you use outbound FQDN rules, charges for Azure Firewall are included in your billing.\n",
+    "\n",
+    "To create a FQDN outbound rule, we will define the `FqdnDestination` Python object with the following attributes:\n",
+    "- `name` - Name of the FQDN outbound rule.\n",
+    "- `destination` - Fully qualified domain name to which outbound connections are allowed, for example: “*.contoso.com”."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ws_mvnet.managed_network.outbound_rules.append(\n",
+    "    FqdnDestination(name=\"fqdnrule\", destination=\"pypi.org\")\n",
```

Comment:
Also add an example with multiple FQDNs in one string, or mention that it's possible.

Reply:
done